br-webui/index.js

```diff
@@ -896,6 +922,32 @@ io.on('connection', function(socket) {
 			socket.emit('video up');
 		});
 		
+		cmd.on('error', (err) => {
+			logger.log('Failed to start child process.');
+			logger.log(err.toString());
+		});
+	});
+
+	socket.on('restart mavproxy', function(data) {
+		logger.log(_companion_directory + '/scripts/restart-mavproxy.sh ' + data.mavOptions);
+		var cmd = child_process.spawn(_companion_directory + '/scripts/restart-mavproxy.sh', {
```

Comment:
data is unused?

Reply:
Ah, that's left over from a previous attempt.  I'll remove it.